corehq/apps/accounting/static/accounting/js/billing_account_form.js

```diff
@@ -9,12 +9,13 @@ hqDefine('accounting/js/billing_account_form', [
     ko,
     initialPageData
 ) {
-    var billingAccountFormModel = function (isActive, isCustomerBillingAccount, enterpriseAdminEmails) {
+    var billingAccountFormModel = function (isActive, isCustomerBillingAccount, enterpriseAdminEmails, isSMSBillableReportVisible) {
         'use strict';
         var self = {};
 
         self.is_active = ko.observable(isActive);
         self.is_customer_billing_account = ko.observable(isCustomerBillingAccount);
+        self.is_sms_billable_report_visible = ko.observable(isSMSBillableReportVisible);
```

Comment:
generally, with camel case, acronyms are lowercase except for the first letter

eg `isSmsBillableReportVisible`